index.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+<title>House Build Project</title>
+
+<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+<p>my house</p>
+
+	<div class="container">
+
+   		<div class="building">

```

Comment:
You have 2 "container" divs. This should be avoided if you can. Consider getting rid of the `.container` div or the `.building` div or combining them into one if you need the classes.
```html
<div class="container building">
```